### DISCUSSION TITLE
Synchronization between databases and SSL

### DISCUSSION THREAD

**Discussion by LLKoder** [keeweb/keeweb#2165] [General Discussion]
I'm looking for a password database keeper that works as a server with a database on a server. I think KeeWeb suit for me if it can sync bases on server itself.
1. I want to ask is there synchronization between databases like KeeShare in KeePassXC?
2. How can I use own SSL certification in KeeWeb?

**Comment by Aetherinox** [comment]
@antelle May know about the KeeShare functionality, I'm new to the development team and am still working my way through the code, but I have not seen anything associated to database sharing yet.

KeeWeb has numerous ways of syncing your database on cloud services such as Google, OneDrive, etc. But if you're looking to set up your own server and use a self-signed certificate; then you're going to want to use the `WebDav` option. Docs are available here: https://github.com/keeweb/keeweb/wiki/WebDAV%20Config

**Reply by antelle** [reply]
No there’s no support for KeeShare in KeeWeb, but can be implemented of course

**Reply by LLKoder** [reply]
I edit post and write more exactly my questions.

As I understand KeeWeb for now can do:
1. Can.
2. I dont undestand can KeeWeb use custom SSL certificat with HTTPS?
3. Not implemented for now, possibly will be implemented in future.
